test/Table_Tests/src/Database/Helpers/Name_Generator.enso

```diff
@@ -0,0 +1,11 @@
+from Standard.Base import all
+
+polyglot java import java.util.UUID
+
+random_uuid : Text
+random_uuid =
+    UUID.randomUUID.to_text
+
+fresh_name : Text -> Text
+fresh_name prefix =
```

Comment:
`unique_name` ? 
Fresh name feels wrong

Reply:
I don't think even that, since this doesn't really guarantee "uniqueness" (would have to compare against some set of names)

Reply:
Isn't the whole idea of UUID that they can generally be treated as unique for such purposes? Of course we can get a collision, but it should be extremely unlikely.

I initially used `fresh` as in generating fresh variable names, for example see [this Racket doc](https://docs.racket-lang.org/redex/reference.html#:~:text=Fresh%20variable%20clauses%20generate%20variables%20that%20do%20not%20occur%20in%20the%20term%20being%20reduced.).

Any other suggestions then? I guess it could be called `random_name`.